diff -Nur synce-sync-engine-0.12.orig/config/org.synce.SyncEngine.service synce-sync-engine-0.12/config/org.synce.SyncEngine.service
--- synce-sync-engine-0.12.orig/config/org.synce.SyncEngine.service	2008-07-16 00:36:43.000000000 +0100
+++ synce-sync-engine-0.12/config/org.synce.SyncEngine.service	2008-10-27 15:26:50.000000000 +0000
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.synce.SyncEngine
-Exec=/usr/bin/sync-engine --detached --logfile=~/.synce/sync-engine.log --once
+Exec=/usr/bin/synce-sync-engine --detached --logfile=~/.synce/sync-engine.log --once
